src/config.ts

```diff
@@ -2913,6 +2926,13 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 				coinGeckoId: 'pstake-finance',
 				coinImageUrl: window.location.origin + '/public/assets/tokens/pstake.png',
 			},
+			{
+				coinDenom: 'g-WBTC',
+				coinMinimalDenom: 'gravity0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599',
+				coinDecimals: 8,
+				coinGeckoId: 'bitcoin',
+				coinImageUrl: window.location.origin + '/public/assets/tokens/btc.png',
```

Comment:
should be '...wtbc.png', and the images replaced with the Wrapped Bitcoin logo

Reply:
Fixed!